addons/mail/models/mail_thread.py

```diff
@@ -2658,12 +2658,15 @@ def _notify_get_action_link(self, link_type, **kwargs):
             'model': kwargs.get('model', self._name),
             'res_id': kwargs.get('res_id', self.ids and self.ids[0] or False),
         }
-        # whitelist accepted parameters: action (deprecated), token (assign), access_token
-        # (view), auth_signup_token and auth_login (for auth_signup support)
+        # keep only accepted parameters:
+        # - action (deprecated), token (assign), access_token (view)
+        # - auth_signup: auth_signup_token and auth_login
+        # - portal: pid, hash
         params.update(dict(
             (key, value)
             for key, value in kwargs.items()
-            if key in ('action', 'token', 'access_token', 'auth_signup_token', 'auth_login')
+            if key in ('action', 'token', 'access_token', 'auth_signup_token',
```

Comment:
Commit:

> [FIX] mail: support portal redirection in _notify_get_action_link

Would it make sense to make this modular, aka in a method that is overridable so that modules can add their own accepted keys?
Or is it something that is deep in the technical code and that we don't want to open too much?

Also, I'm guessing this change doesn't do anything on its own, it just enables more features support for the future?